### DISCUSSION TITLE
Async SQL (Relational) Databases example in Docs is broken

### DISCUSSION THREAD

**Discussion by JamesStuff** [fastapi/fastapi#8400] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
website = "https://fastapi.tiangolo.com/tutorial/"

copy_tutorial_code()

result = """
...
ValueError: not enough values to unpack (expected 5, got 4)
"""
```

### Description

Following the tutorial code for Async Databases https://fastapi.tiangolo.com/advanced/async-sql-databases/ results in an error when trying to view all records of a field, in this case preforming the GET operation on all notes. it will error on the line `query = notes.select()`

### Operating System

Linux, macOS

### Operating System Details

Ubuntu 22.04.1 x86 12GB Ram on VM
macOS 13 (ARM64 M1 Max) 

### FastAPI Version

0.85.1

### Python Version

Python 3.10.6

**Comment by PabloCastellano** [comment]
The problem exists with [SQLalchemy 1.4.42](https://github.com/sqlalchemy/sqlalchemy/releases/tag/rel_1_4_42). Downgrading to 1.4.41 solved the issue for me.

**Comment by iudeen** [comment]
@PabloCastellano and @JamesStuff for better context to future visitors to this issue, can you add a MRE?

**Comment by PabloCastellano** [comment]
@iudeen What does MRE stand for?

**Comment by iudeen** [comment]
Minimal reproducible example.

**Comment by musicinmybrain** [comment]
https://github.com/tiangolo/fastapi/pull/5799
